access-functions.php

```diff
@@ -202,3 +202,12 @@ function register_graphql_mutation( $mutation_name, $config ) {
 		$type_registry->register_mutation( $mutation_name, $config );
 	} );
 }
+
+/**
+ * Whether a GraphQL request is in action or not. Default false.
+ *
+ * @since 0.4.1
+ */
+function is_graphql_request() {
```

Comment:
Should `is_http_graphql_request()` to be exposed too? Although `is_graphql_request` should be generally preferred.

Reply:
Ah yes. I think both make sense to have. I do think there is value in being able to determine both cases

Reply:
added